FrenchFilter/sections/antiadblock.txt

```diff
@@ -576,10 +579,13 @@ prod-player.tf1.fr,tf1.fr,tf1info.fr#@#.carbon-ads
 @@||tf1info.fr^$generichide
 !#endif
 !
-! (2/2) "NETWORK REQUEST" AAB CHECK
+! (3/3) "NETWORK REQUEST" AAB CHECK
 !
-tf1.fr#%#AG_defineProperty('__TF1_CONFIG__.adblock.serverRequest', { value: false, writable: false });
-tf1info.fr#%#AG_defineProperty('__NEXT_DATA__.runtimeConfig.adBlock.serverRequest', { value: false, writable: false });
+tf1.fr#%#//scriptlet('set-constant', '__TF1_CONFIG__.adblock.serverRequest', 'false')
+tf1info.fr#%#//scriptlet('set-constant', '__NEXT_DATA__.runtimeConfig.adBlock.serverRequest', 'false')
+! https://github.com/AdguardTeam/AdguardFilters/issues/141901#issuecomment-1434658668
+tf1.fr,tf1info.fr#%#//scriptlet('prevent-fetch', 'static.adsafeprotected.com/favicon.ico method:HEAD')
+tf1.fr,tf1info.fr#%#//scriptlet('prevent-fetch', '/^https:\/\/ads\.stickyadstv\.com\/$/ method:HEAD')
```

Comment:
is `method` required here?

Reply:
Yes, ideally. I mean that, in all cases (even for the `favicon.ico` request, see for example [here for tf1info.fr](https://user-images.githubusercontent.com/4764956/220398649-f3b3bbba-d978-4235-ad67-42806cef726f.png)), they only do HEAD requests for AAB checks:
- [tf1.fr](https://user-images.githubusercontent.com/4764956/220392899-f6f6a23e-1a57-4eda-a7dc-ab8c1a507095.png)
- [tf1info.fr](https://user-images.githubusercontent.com/4764956/220392946-c215b44f-0d6d-465c-b575-d637dd45985e.png)

_(+ confirmed in Firefox "Developer Tools" > "Network" tab as shown above and all tested. :thumbsup:)_

Reply:
Ok, thanks.

Reply:
No problem. Thank _you_! 🙂